lte/gateway/python/magma/pipelined/qos/common.py

```diff
@@ -108,7 +108,7 @@ def remove_session(self, ip_addr: str) -> None:
         del self.sessions[ip_addr]
 
     def find_session_with_rule(self, rule_num: int) -> SubscriberSession:
-        session_with_rule = None
+        session_with_rule = SubscriberSession()
```

Comment:
I think it would be better to change the return value to `Optional[SubscriberSession]` here. Returning a new session seems strange.